src/components/MapView/Layers/BoundaryDropdown.tsx

```diff
@@ -37,12 +48,31 @@ const ClickableListSubheader = styled(ListSubheader)(({ theme }) => ({
     backgroundColor: theme.palette.grey[100],
   },
 }));
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme: Theme) => ({
   searchField: {
     '&>div': {
       color: 'black',
     },
   },
+  dropdownMenu: {
+    backgroundColor: theme.palette.primary.main,
+    color: 'white',
+    display: 'inline-flex',
+    marginTop: '1em',
+    padding: '6px 16px',
```

Comment:
try use `theme.spacing(1,2)` when possible

Reply:
included in cfa705fe46213f209d51a49cef06e81a6ad3a0c8